data/map planets.txt

```diff
@@ -2649,6 +2708,15 @@ planet Memory
 		threshold 4000
 		fleet "Small Deep Security" 16
 
+planet "Mendez Station"
+	attributes CCOR station
+	landscape land/station20
+	description 'Mendez Station is the closest object to Alpherg's twin stars. Mostly uninhabited Mendez sits in low orbit using it's monumental ramscoops to collect solar wind to process into fuel. The station is uncomfortably hot, life support systems only turning on and beginning to cool the habitation segment when a ship is docked. The refining process here is almost entirely automated, the habitation segment was only expected to be in use during routine inspections and maintenance.'
+	spaceport 'Forgoing a traditional docking ring, which would be excessive for such a low traffic station, Mendez's 'spaceport' is a small number of almost archaic docking arms leading to the hab segment. There was a security system built into each airlock, engraved with the seal of the CCOR government, but they've all been either smashed, bypassed, or hacked allowing unrestricted access to the station.'
```

Comment:
```suggestion
```

Reply:
> What I want to avoid (which is happening currently due to a spaceport) is a bunch of Pirates landing on it and taking off from it, which is implying it's a busy station that accommodates common visitors  or is publicly active. It collects fuel close to the star, but I wouldn't say it's a refuelling outpost? Isn't it moreso a collection station and not hub? 

> Since we are saying it's a smaller sattelite station in order for it to be orbiting the star, it probably wouldn't be efficient at accommodating common Pirates with fuel if that were the case. So yeah I'd remove the spaceport. You can still have missions offer from it of course, just though landing.